Gesture_control.py

```diff
@@ -8,6 +8,34 @@
 import cv2
 import numpy as np
 
+
+def get_trackbars():
+    cv2.namedWindow('Trackbars')
+    cv2.resizeWindow('Trackbars', 640, 240)
+    cv2.createTrackbar('HueMin', 'Trackbars', 0, 179, empty)
+    cv2.createTrackbar('HueMax', 'Trackbars', 0, 179, empty)
+    cv2.createTrackbar('SatMin', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('SatMax', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('ValMin', 'Trackbars', 0, 255, empty)
+    cv2.createTrackbar('ValMax', 'Trackbars', 0, 255, empty)
+
+def create_mask():
+    while True:
+        img = cv2.imread('') # read the current frame
+        imgHSV = cv2.cvtColor(img, cv2.COLOR_BGR2HSV)
+        hue_min = cv2.getTrackbarPos('HueMin', 'Trackbars')
+        hue_max = cv2.getTrackbarPos('HueMax', 'Trackbars')
+        sat_min = cv2.getTrackbarPos('SatMin', 'Trackbars')
+        sat_max = cv2.getTrackbarPos('SatMax', 'Trackbars')
+        val_min = cv2.getTrackbarPos('ValMin', 'Trackbars')
+        val_max = cv2.getTrackbarPos('ValMax', 'Trackbars')
+        lower = np.array([hue_min, sat_min, val_min])
+        upper = np.array([hue_max, sat_max, val_max])
+        mask = cv2.inRange(imgHSV, lower, upper)
```

Comment:
Even after implementing all above changes, you have no provision to access the `mask` variable outside the function.